fastly/block_fastly_service_director_test.go

```diff
@@ -114,6 +114,11 @@ func TestResourceFastlyFlattenDirectors(t *testing.T) {
 	}
 }
 
+// This test validates that two directors are created successfully,
+// and in the next Terraform run the first director is updated while
+// the second director is unchanged and a third director is added.
+// In the final test, the first director is removed while the second
+// director is unchanged and one backend for the third director is removed.
 func TestAccFastlyServiceVCL_directors_basic(t *testing.T) {
```

Comment:
❤️ 

I always appreciate when people take the time to provide a 'summary' of a test.